### PR TITLE
4 feature account class creation

### PR DESCRIPTION
**Pull request by AlanC12138**
added text file for storing user credentials, removed gpa methods from archive and runningcourses; added more comments; modified functions in Accountlist to fit with the new textfile

### CODE REVIEW

**Review by YuweiCai2003 — DISMISSED** [review]
Separation into different classes makes the code really efficient

**Review by berrythebear — COMMENTED** [review]
Uses of the filepath will have to be modified to work for any user. (new File(System.getProperty("user.dir"), "src\\main\\java\\accountClasses\\accountStorage") should work. Also, mkdirs() can create parts of the directory that don't yet exist such as if accountStorage is a folder. ".txt" might be needed at the end if you're using a text file to store the information

**Review by berrythebear — DISMISSED** [review]

File: src/test/java/courseManager/CourseManagerTest.java [hunk 1]
```diff
@@ -1,8 +1,10 @@
 package courseManager;
 
+import entities.assessment.Assessment;
+import entities.course.Course;
```
Comment (MrSunshineVS) [inline src/test/java/courseManager/CourseManagerTest.java:4]: This file is behind main. sami's pull request was approved recently, try merging main and hopefully the issue will be resolved

**Review by MrSunshineVS — CHANGES_REQUESTED** [review]
Everything looks great!!!!!! I think the test file in your branch is behind main though so try fixing that before we merge.

**Review by MrSunshineVS — APPROVED** [review]

**Pull request closed without merging**